tests/IsoCodes/Tests/AbstractIsoCodeTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace IsoCodes\Tests;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+abstract class AbstractIsoCodeTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @return array[]
+     */
+    abstract public function getValidValues();
+
+    /**
+     * @return array[]
+     */
+    abstract public function getInvalidValues();
+
+    final public function testEmptyValues()
```

Comment:
Wrong, this is a `test*` method, so each test class extending `AbstractIsoCodeTest` will do the `testEmptyValues` on it.

Just put a `die` inside the function, you will see! :+1: